README.md

```diff
@@ -37,17 +42,34 @@ Using Maven:
 </dependency>
 ```
 
+### For Spark 2.0
+
+Using SBT:
+
+```
+libraryDependencies += "com.databricks" %% "spark-avro" % "3.0.0"
```

Comment:
The `%%` will add the `_2.10` or `_2.11` based on your scala version of choosing